src/cloud/cdn/configure-fastly.md

```diff
@@ -185,6 +185,9 @@ To update your DNS configuration for development:
 
    When the CNAME records are live, Magento provisions certificates and uploads the SSL/TLS certificates.
 
+   {:.bs-callout-info}
+   If you plan to use apex domains (`your-domain.com`) for your Production environment, you must configure A records to point to the Fastly server IP addresses. See [Update DNS configuration with production settings]({{ site.baseurl }}/cloud/live/site-launch-checklist.html#dns).
```

Comment:
```suggestion
   If you plan to use apex domains (`your-domain.com`) for your Production environment, you must configure `A records` to point to the Fastly server IP addresses. See [Update DNS configuration with production settings]({{ site.baseurl }}/cloud/live/site-launch-checklist.html#dns).
```

Reply:
If `A records` is not highlighted like this elsewhere, you can ignore this. I just tripped on it at first.